src/main/scala/zio/akka/cluster/sharding/Sharding.scala

```diff
@@ -136,12 +136,13 @@ object Sharding {
     val ref: Ref[Option[State]]    = rts.unsafeRun(Ref.make[Option[State]](None))
     val actorContext: ActorContext = context
     val entity: ZLayer[Any, Nothing, Entity[State]] = ZLayer.succeed(new Entity.Service[State] {
+      override def context: Task[ActorContext]                   = Task(actorContext)
```

Comment:
should this be a `ZIO`, since the context is already created it shouldn't fail anymore.